### PR TITLE
Cleanup for about fluentbit and fluentd

### PR DESCRIPTION
**Pull request by esmerel**
Standardizing this page for style.

Note: Line 36 was confusing to me. I rewrote it to something that makes sense, but I'm also not completely sure I'm correct.

### CODE REVIEW

File: about/fluentd-and-fluent-bit.md [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
-| ------------ | ----------------------------------------------------------------- | ----------------------------------------------------------------- |
-| Scope        | Containers / Servers                                              | Embedded Linux / Containers / Servers                             |
-| Language     | C & Ruby                                                          | C                                                                 |
-| Memory       |  > 60MB                                                           | \~1MB                                                             |
-| Performance  | Medium Performance                                                | High Performance                                                  |
-| Dependencies | Built as a Ruby Gem, it requires a certain number of gems.        | Zero dependencies, unless some special plugin requires them.      |
-| Plugins      | More than 1000 external plugins are available                     | More than 100 built-in plugins are available                      |
+|              | Fluentd               | Fluent Bit            |
```
Comment (cnorris-cs) [inline about/fluentd-and-fluent-bit.md:30]: We should have a title for this column. "Attribute", perhaps?

File: about/fluentd-and-fluent-bit.md [hunk 2]
```diff
@@ -1,33 +1,45 @@
...
+| ------------ | --------------------- | --------------------- |
+| Scope        | Containers / Servers  | Embedded Linux / Containers / Servers |
+| Language     | C & Ruby              | C                                     |
+| Memory       |  > 60MB               | ~1MB                                  |
```
Comment (cnorris-cs) [inline about/fluentd-and-fluent-bit.md:34]:
```suggestion
| Memory       |  Greater than 60&nbsp;MB | Approximately 1&nbsp;MB |
```

File: about/fluentd-and-fluent-bit.md [hunk 3]
```diff
@@ -1,33 +1,45 @@
...
+| Performance  | Medium Performance    | High Performance                      |
+| Dependencies | Built as a Ruby Gem, depends on other gems. | Zero dependencies, unless required by a plugin. |
+| Plugins      | Over 1000 external plugins available. | Over 100 built-in plugins available. |
```
Comment (cnorris-cs) [inline about/fluentd-and-fluent-bit.md:37]:
```suggestion
| Plugins      | Over 1,000 external plugins available. | Over 100 built-in plugins available. |
```

File: about/fluentd-and-fluent-bit.md [hunk 4]
```diff
@@ -1,33 +1,45 @@
...
-* Widely Adopted by the Industry: trusted by all major companies like AWS, Microsoft, Google Cloud and hundreds of others.
+Both projects are:
 
-Both projects share a lot of similarities, [Fluent Bit](https://fluentbit.io) is fully designed and built on top of the best ideas of [Fluentd](https://www.fluentd.org) architecture and general design. Choosing which one to use depends on the end-user needs.
+- Licensed under the terms of Apache License v2.0.
+- Graduated hosted projects by the [Cloud Native Computing Foundation (CNCF)](https://cncf.io).
+- Production grade solutions: Deployed **million** of times every single day.
+- Vendor neutral and community driven.
+- Widely adopted by the industry: Trusted by major companies like AWS, Microsoft,
+  Google Cloud and hundreds of others.
```
Comment (cnorris-cs) [inline about/fluentd-and-fluent-bit.md:22]:
```suggestion
  Google Cloud, and hundreds of others.
```

File: about/fluentd-and-fluent-bit.md [hunk 5]
```diff
@@ -1,33 +1,45 @@
...
+
+The projects have many similarities: [Fluent Bit](https://fluentbit.io) is
+designed and built on top of the best ideas of [Fluentd](https://www.fluentd.org)
+architecture and general design. Choosing one depends on end-user needs.
```
Comment (cnorris-cs) [inline about/fluentd-and-fluent-bit.md:26]:
```suggestion
architecture and general design. Which one you choose depends on your end-users' needs.
```

**Review by cnorris-cs — APPROVED** [review]
I left suggestions, but I've added my approval based on that to keep this moving.

(And, yes, I know I made some additional changes. This is just the first page we've updated like this. I'll be able to focus more on just the basic changes as we go.)

File: about/fluentd-and-fluent-bit.md [hunk 6]
```diff
@@ -1,33 +1,45 @@
...
-* **Graduated** Hosted projects by the [Cloud Native Computing Foundation (CNCF)](https://cncf.io)
-* Production Grade solutions: deployed **million** of times every single day.
-* **Vendor neutral** and community driven projects
-* Widely Adopted by the Industry: trusted by all major companies like AWS, Microsoft, Google Cloud and hundreds of others.
+Both projects are:
 
-Both projects share a lot of similarities, [Fluent Bit](https://fluentbit.io) is fully designed and built on top of the best ideas of [Fluentd](https://www.fluentd.org) architecture and general design. Choosing which one to use depends on the end-user needs.
+- Licensed under the terms of Apache License v2.0.
+- Graduated hosted projects by the [Cloud Native Computing Foundation (CNCF)](https://cncf.io).
+- Production grade solutions: Deployed **million** of times every single day.
```
[the remark this answers is not in the document]
Comment (esmerel) [reply about/fluentd-and-fluent-bit.md:19]: except the emphasis also should be removed heh

[the remark this answers is not in the document]
Comment (esmerel) [reply about/fluentd-and-fluent-bit.md:19]:
```suggestion
- Production grade solutions: Deployed millions of times every single day.
```

File: about/fluentd-and-fluent-bit.md (hunk 1 above)
Comment (esmerel) [reply about/fluentd-and-fluent-bit.md:30]: in theory, but I thought that might be confusing with the other two columns being products and also whether that was out of scope for general cleanup.

**Pull request merged**